editor/components/toolbar/index.js

```diff
@@ -28,7 +28,7 @@ function Toolbar( { controls } ) {
 					} }
 					className={ classNames( 'editor-toolbar__control', {
 						'is-active': control.isActive
-					} ) } />
+					}, control.classNames ) } />
```

Comment:
>It could be useful to leave for others to be able to tweak controls.

We can always reintroduce it if it proves to be useful, but I'm generally against introducing functionality we expect _could_ be useful but not actually use ([YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)).

If and when it can be shown to provide value, I think there could also be a discussion on the naming and shape of `classNames`. Is it a `className` (singular) string? A `classNames` (plural) array of strings?